lesson18/main.js

```diff
@@ -94,22 +92,35 @@ const createBackBtn = () => {
   backBtn.addEventListener(
     "click",
     () => {
-      switchImg("previousElementSibling");
-      switchDisableForBtn();
-      getCurrentPageNum();
+      clickBtnEvents("previousElementSibling");
     },
     false
   );
 };
 
-const  switchImg = (SwitchDirection) => {
+const clickBtnEvents = (SwitchDirection) => {
+  switchImgForBtn([SwitchDirection]);
+  switchDotForBtn([SwitchDirection]);
+  switchDisableForBtn();
+  getCurrentPageNum();
+};
+
+const switchImgForBtn = (SwitchDirection) => {
   const currentImg = document.querySelector(".is-show");
   if (currentImg[SwitchDirection]) {
     currentImg.classList.remove("is-show");
     currentImg[SwitchDirection].classList.add("is-show");
   }
 };
 
+const switchDotForBtn = (SwitchDirection) => {
```

Comment:
```suggestion
const changeIndicator = (SwitchDirection) => {
```

Reply:
I realized that the name is definitely less conservative. I fixed.
[e2ed8ce](https://github.com/yuka830/js-lessons/pull/23/commits/e2ed8ce7b8ff7bda24057c31864762686d15e8ab)

And also I used 'dot' for other functions and class name. I changed them too.
[8febe0d](https://github.com/yuka830/js-lessons/pull/23/commits/8febe0df2b9a3a92c9d8fa0a69701f0f08653c44)